### PR TITLE
Address flaky test KeepAliveHandlerTest.testIdle

### PR DESCRIPTION
**Pull request by jrhee17**
Motivation:

Address flaky test KeepAliveHandlerTest.testIdle

Modifications:

- Wait for meters to be recorded before performing assertions on them.

Result:

- Feel free to close this PR if you had a better idea in mind @ikhoon 🙏 🙇

### CODE REVIEW

File: core/src/test/java/com/linecorp/armeria/internal/common/KeepAliveHandlerTest.java [hunk 1]
```diff
@@ -121,6 +121,11 @@ protected boolean hasRequestsInProgress(ChannelHandlerContext ctx) {
 
         idleTimeoutScheduler.initialize(ctx);
         await().timeout(4, TimeUnit.SECONDS).untilAtomic(counter, Matchers.is(1));
+
+        // add another listener to ensure all closeFuture listeners are invoked at this point
+        ctx.channel().closeFuture().addListener(unused -> counter.incrementAndGet());
```
Comment (ikhoon) [inline core/src/test/java/com/linecorp/armeria/internal/common/KeepAliveHandlerTest.java:126]: Awesome! Nothing better than this. 😆

**Review by ikhoon — APPROVED** [review]
💯

**Review by minwoox — APPROVED** [review]
👍 👍 👍

**Review by trustin — APPROVED** [review]
Thanks, @jrhee17 !

**Pull request merged**